src/app/containers/Include/index.test.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import IncludeContainer from '.';
+import { ToggleContext } from '#contexts/ToggleContext';
+import { RequestContextProvider } from '#contexts/RequestContext';
+
+const fakeMarkup = `&lt;div&gt;Visual Jounalism Markup&lt;/div&gt;&lt;script type=&quot;text/javascript&quot; src=&quot;localhost/vj.js&quot;&gt;&lt;/script&gt;`;
```

Comment:
```suggestion
const fakeMarkup = `&lt;div&gt;Visual Journalism Markup&lt;/div&gt;&lt;script type=&quot;text/javascript&quot; src=&quot;localhost/vj.js&quot;&gt;&lt;/script&gt;`;
```

I know it's only fixture data 🤷‍♂